docs/gcloud-auth.rst

```diff
@@ -58,15 +58,66 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful
```

Comment:
I don't think semantic breaks are primarily about length... I think the idea is about breaking where phrases end, so the future changes can be focused on the actual thing you're changing. We all have pretty big screens, so more than 80 characters is nice but not going to kill anyone -- it's just the standard now.

Here's an example: https://raw.githubusercontent.com/brandon-rhodes/blog/master/texts/brandon/2012/one-sentence-per-line.rst

The code equivalent of "just fill up to 80 characters" might be....

```python
import foo; foo.method(); i = 0; if foo.method2() and foo.method3(): while i < 5: i++; print "ok";
```

Perfectly valid Python, but it's more than one "line" on a single "line"... Not saying this is an amazing analogy, but not totally useless either.

We break our code when the conceptual statement is done, so I have trouble understanding why we aim to get as close to 80 characters as possible with textual documentation. Even one-sentence-per-line would be a better than "make it as close to an 80-character-wide rectangle as possible".

Why not break it where the thought ends... (typically with a `.` or ` ,` -- you might have to think back to elementary school where we learned how to diagram sentences...).